src/types/products.ts

```diff
@@ -0,0 +1,12 @@
+export interface IProduct {
```

Comment:
Probably use should setup extra rule in eslint/prettier to have a constant indentation across all files.
In this file it's 4 spaces, but in other places 2 spaces.

Reply:
Done. Everywhere 4 indents. added rule for typescript/indent